service/github.ts

```diff
@@ -0,0 +1,36 @@
+import fetch from 'node-fetch';
+
+export async function searchGitHubUsers(params: {
+  searchTerm: string;
+  page: number;
+  perPage: number;
+  sort: string;
+  order: 'asc' | 'desc';
+}) {
+  const { searchTerm, page, perPage, sort, order } = params;
```

Comment:
Please do not move github.client functions here, instead you should call the functions from here.